library/aes.c

```diff
@@ -352,11 +355,12 @@ static const uint32_t RT3[256] = { RT };
 
 #endif /* !MBEDTLS_AES_FEWER_TABLES */
 
-#endif /* !defined(MBEDTLS_AES_DECRYPT_ALT) || !defined(MBEDTLS_AES_SETKEY_DEC_ALT) */
+#endif \
```

Comment:
(aside) it seems a bit weird having a line-continuation character to add a comment that will be removed at an earlier stage of processing anyway

Reply:
That needs to be submitted as a request/patch to uncrustify.